### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
- Biboumi package from Buster repo

[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/biboumi_ynh%20PR7%20(ericgaspar)/)

### CODE REVIEW

**Comment by pitchum** [comment]
Currently I have no way to test your changes but it looks good to me.
Do you consider your PR ready to merge?

**Comment by ericgaspar** [comment]
This is not ready yet... the install fails on my system... :/ 
the logs:
https://paste.yunohost.org/raw/jabasatexe

**Review by alexAubin — APPROVED** [review]
(LGTM so far)

File: scripts/upgrade [hunk 1]
```diff
@@ -28,9 +29,10 @@ upgrade_type=$(ynh_check_app_version_changed)
 #=================================================
 # ENSURE DOWNWARD COMPATIBILITY
 #=================================================
+ynh_script_progression --message="Ensuring downward compatibility..." --weight=1
 
 # Prevent conflict with other apps on the same domain
-ynh_app_setting_set $app no_sso 1
+ynh_app_setting_set --app=$app --key=no_sso --value=1
```
Comment (alexAubin) [inline scripts/upgrade:35]: I believe this setting does not do anything anymore (it was a small hack before the new permission system was introduced)

File: scripts/install [hunk 2]
```diff
@@ -117,18 +114,24 @@ ynh_use_logrotate /var/log/biboumi/
...
-yunohost service add biboumi --log "/var/log/biboumi/biboumi.log"
+yunohost service add biboumi --description "XMPP gateway for the IRC network" --log "/var/log/biboumi/biboumi.log"
 
 #=================================================
 # SET THE APP AS NOT USING THE SSO
 #=================================================
 
 # Prevent conflict with other apps on the same domain
-ynh_app_setting_set $app no_sso 1
+ynh_app_setting_set --app=$app --key=no_sso --value=1
```
Comment (alexAubin) [inline scripts/install:124]: (c.f. other comment)

**Review by alexAubin — APPROVED** [review]
LGTM apart from the no_sso setting which i believe is obsolete

**Pull request merged**